src/tailscale/tailscaled-devcontainer-start.sh

```diff
@@ -31,11 +36,18 @@ TAILSCALED_PID=""
 TAILSCALED_SOCK=/var/run/tailscale/tailscaled.sock
 TAILSCALED_LOG=/var/log/tailscaled.log
 
-# Note: TS_DEBUG_FIREWALL_MODE: it is not recommended that users copy this
-# setting into other environments, the feature is in test and will be formally
-# released in the future, debug flags may later be recycled for other purposes
-# leading to unexpected behavior.
->$TAILSCALED_LOG 2>&1 TS_DEBUG_FIREWALL_MODE=auto /usr/local/sbin/tailscaled &
+(
```

Comment:
Do we want/need to do this step if we're not going to do the `tailscale up` below?

Reply:
yes, this starts tailscaled. I was in the process of trying to make the test better, and there's a race with the way the tests are run in `@devcontainers/cli`, so I did some work to ensure the tailscaled process gets daemonized and doesn't retain any session resources from the parent, but we still want to daemonize the tailscaled in all cases - this is just a more complete daemonization step.